.changeset/calm-goats-talk.md

```diff
@@ -0,0 +1,55 @@
+---
+"@biomejs/biome": minor
+---
+
+Added the `useIframeTitle` lint rule for HTML. The rule enforces the usage of the `title` attribute for the `iframe` element.
+
+In `.html` files, this rule matches `iframe` elements case-insensitively (e.g., `<IFRAME>`, `<IFrame>`).
+
+In component-based frameworks (Vue, Svelte, Astro), only lowercase `<iframe>` is checked. PascalCase variants like `<Iframe>` are assumed to be custom components and are ignored.
```

Comment:
nit: I would even go as far to say that we could omit these lines too. It's an implementation detail.